Sources/TokamakCore/Views/Image.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018-2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Jed Fox on 07/01/2020.
+//
+
+import Foundation
+
+public struct Image: View {
```

Comment:
I think to match some of SwiftUI's `Image` modifiers we'll need to implement the internal storage as an `AnyTokenBox`. Then once we add support for things like template images we can access the environment to render them.

Reply:
Thanks for highlighting this, I've created https://github.com/TokamakUI/Tokamak/issues/312 to track it.